securedrop/config.py.example

```diff
@@ -5,6 +5,8 @@ class FlaskConfig(object):
     DEBUG = False
     TESTING = False
     WTF_CSRF_ENABLED = True
+    USE_X_SENDFILE = False
+    MAX_CONTENT_LENGTH = 524288000
```

Comment:
This one is consistent with production but it would need to be kept in sync with the value from `install_files/ansible-base/roles/app/templates/sites-available/journalist.conf`  and `install_files/ansible-base/roles/app/templates/sites-available/source.conf`. This is problematic because there is no update path for this value in the config.py file.

For instance imagine we raise that value in the apache files to 1GB. It will be updated via `./securedrop-admin ansible` but ... the value in config.py stays 500MB and the change has no effect.

I'm afraid we'll have to find another way to make mypy happy.

Reply:
Btw, those values are from the current config.py.example itself.

Reply:
disabling the check for this file seems like a good idea